src/instructlab/model/backends/vllm.py

```diff
@@ -207,11 +210,6 @@ def format_template(model_family: str, model_path: pathlib.Path) -> str:
     return prefix + template
 
 
-def contains_argument(prefix: str, args: typing.Iterable[str]) -> bool:
```

Comment:
This is also used from serve and evaluate

Reply:
good catch, thanks! updated

I'm surprised the import didn't break - seems like python may be able to handle transitive imports somehow?